src/staticanalysis/bot/static_analysis_bot/revisions.py

```diff
@@ -178,6 +179,9 @@ def load(self, repo):
             # Use base revision from top diff
             hg_base = self.diff['baseRevision']
 
+        if hg_base is None:
+            raise AnalysisException('phabricator', 'Missing base revision')
```

Comment:
We should not fail in this case, but try to apply the patch to latest mozilla-central. Maybe just log a warning here